README.md

```diff
@@ -150,10 +150,12 @@ Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com
 
 ## Communication
 
+- [GitHub Discussions] for feature requests and questions.
```

Comment:
I didn't know that we opened the GitHub discussions, but users have already communicates each other on it 👍 . BTW, Both `GitHub Discussions` and `GItHub Issues` have `feature requests` and `questions`, and users might get confused. So, we may need to discuss the policy of these tools before adding these sentences.

Reply:
We discussed how we should use GitHub Discussions at the Optuna core development team MTG. And then we decided to use Github Discussions as a place where users can casually post questions, and GitHub issues will continue to be used for bug reports and feature requests.

Do you think it makes sense?

Reply:
So we might have to update Optuna's issue template as well.
https://github.com/optuna/optuna/blob/master/.github/ISSUE_TEMPLATE/questions-help-support.yml

FYI: In Optuna Dashboard, I prepared an issue template to prevent users to ask support questions.
https://github.com/optuna/optuna-dashboard/blob/main/.github/ISSUE_TEMPLATE/question.md

Reply:
@c-bata thank you for sharing the discussion. Yes, I think it makes sense.